python/lsst/afw/cameraGeom/detectorCollection.py

```diff
@@ -20,6 +20,8 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 from __future__ import absolute_import, division
+from past.builtins import basestring
```

Comment:
No. Because of this:
```
>>> from builtins import str
>>> from past.builtins import basestring
>>> isinstance("xx", str)
False
>>> isinstance("xx", basestring)
True
>>> isinstance(str("xx"), basestring)
True
```